File: app/controllers/statistics_controller.rb
class StatisticsController < ApplicationController
  def show
    @most_popular_house = House.left_joins(:bookings).group(:id).order('count(bookings.id) DESC').first

    @most_successful_year = Booking.group("strftime('%Y', day)").order('count(day) DESC').limit(1).count.keys.first
  end
end

File: src/Controllers/StatisticsController.php
<?php

namespace BookMyHouse\Controllers;

use BookMyHouse\Models\House;

class StatisticsController extends BaseController
{
    public function show(): void
    {
        $houses = House::all();
        $mostPopularHouse = !empty($houses) ? $houses[0] : null;
        $mostSuccessfulYear = 2014;

        $this->render('statistics/show', [
            'most_popular_house' => $mostPopularHouse,
            'most_successful_year' => $mostSuccessfulYear
        ]);
    }
}